bin/elasticsearch.bat

```diff
@@ -1,10 +1,46 @@
 @echo off
 
-SETLOCAL
+SETLOCAL enabledelayedexpansion
 TITLE Elasticsearch ${project.version}
 
+SET params='%*'
+
+:loop
+FOR /F "usebackq tokens=1* delims= " %%A IN (!params!) DO (
+    SET current=%%A
+    SET params='%%B'
+	SET silent=N
+	
+	IF "!current!" == "-s" (
+		SET silent=Y
+	)
+	IF "!current!" == "--silent" (
+		SET silent=Y
+	)	
+	
+	IF "!silent!" == "Y" (
+		SET nopauseonerror=Y
+	) ELSE (
```

Comment:
I did not mean that. Sorry if I was unclear. I was just trying to avoid rewriting users arguments to `elasticsearch.bat`. So if a user run `elasticsearch --silent -Des.cluster.name=toto` it will run behind the scene `java -cp "%ES_CLASSPATH%" "org.elasticsearch.bootstrap.Elasticsearch" --silent -Des.cluster.name=toto` but `--silent` will be totally ignored by elasticsearch...

Not a big deal to me. Just trying to have less code to maintain :)